packages/config/src/bin/build-script.ts

```diff
@@ -23,9 +23,10 @@ export function buildScript(argv: { [name: string]: string }, cwd: string) {
     fecLogger(LogType.error, err);
     process.exit(1);
   });
-  subprocess.on('exit', (code: string | null, signal: string) => {
+  subprocess.on('exit', (code: number | null, signal: string) => {
```

Comment:
This looks more logical and also according to https://nodejs.org/api/child_process.html#event-exit it seems correct